test/unit/au/org/emii/aatams/ReceiverDeploymentTests.groovy

```diff
@@ -4,22 +4,34 @@ import java.util.Date
 import java.util.Set
 
 import org.joda.time.DateTime
+import org.joda.time.Interval
 
 import com.vividsolutions.jts.geom.Point
 import com.vividsolutions.jts.io.WKTReader
 
 import grails.test.*
 
-class ReceiverDeploymentTests extends GrailsUnitTestCase
-{
-    void testScheduledRecoveryDateBeforeDeploymentDate()
-    {
+class ReceiverDeploymentTests extends GrailsUnitTestCase {
+    static def now = new DateTime()
+
+    void setUp() {
+        super.setUp()
+
+        JodaOverrides.apply()
+        mockDomain(ReceiverDeployment)
+    }
+
+    void tearDown() {
+        JodaOverrides.unmock()
```

Comment:
In the setup() we used apply() (not mock()). Do we still need to do this (unmock())?

ALL THE PARENTHESES!!!!!!!!11